practices/icebreaker.md

```diff
@@ -0,0 +1,95 @@
+---
+slug: icebreaker
+stages:
+  - development
+short_description: Icebreaker is an activity used at the beginning of a meeting to attendees more comfortable with each other, encouraging them to speak up, listen up, join in, and open up to new ideas.
+tags:
+  - meaningful meetings
+  - meetings
+  - event
+  - rituals
+  - agile events
+  - scrum
+  - retrospective
+  - coaching
+  - leadership
+  - feedback
+  - good developer experience
+  - demotivated team
+keywords:
+  - icebreaker
+  - activity
+  - game
+  - break the ice
+  - team building
+  - communication
+  - networking
+  - experience exchange
+  - remote activity
+---
+
+# Icebreaker
+
+**TL;DR**
+
+Icebreaker is an activity used at the beginning of a meeting to attendees more comfortable with each other, encouraging them to speak up, listen up, join in, and open up to new ideas.
+
+
+## What Is an Icebreaker
+
+An icebreaker is an activity or game designed to "break the ice" between attendees in a meeting. In general, it is used at the beginning of the session to introduce participants to each other, get them relaxed and energized for the upcoming activities. 
+
+Icebreakers are unrestricted to the beginning of a meeting, neither to a random group of attendees. They work very well at the start, middle (after a break), and at the end of a session, and also work very well for attendees who are familiar with each other because they help to set a safe space for attendees to learn and share.
+
+## Why You Might Want Use Icebreakers
+
+Icebreakers are fun and remarkable ways to start meetings or training sessions. They are a versatile and adaptable tool used to make the attendees more comfortable with each other, encouraging them to speak up, listen up, join in, and open up to new ideas.
+
+These activities foster networking and a sense of community due to the experience exchange. They can set the tone for the upcoming session and give participants ownership of the learning, with no pressure or high expectations.
+
+## Problems Icebreakers Help to Solve
+
+- Higher anxiety and stress levels pre-meeting
+- Low interaction with unknown colleagues
+- Low engagement to speak up
+- [Demotivated team](/problems/demotivated-team)
+- Decrease in energy levels during a long meeting
```

Comment:
```suggestion
- Decrease in energy levels during a [meaningful meeting](practices/meaningful-meetings)
```

Reply:
I added 
```suggestion
- Decrease in energy levels during non [meaningful meetings](practices/meaningful-meetings)
```
because we are talking about the **problems** icebreakers help to solve. What do you think about this approach?